### ISSUE TITLE
How to built ctc_topo with blankid != 0

### ISSUE THREAD

**Issue by Slyne** [k2-fsa/snowfall#249]
Hi experts,
I was wondering if the build_ctc_topo interface can be used for blankid is not equal to 0?
What's the correct way to build the ctc_topo in this situation? My model has a blankid=1024.

I believe the root cause of this question is I don't understand how we map the logits/probabilities of vocabulary to the arcs in T.fst. If you can help explain, I would be really grateful.

Thank you!!
